apps/src/p5lab/spritelab/commands/spriteCommands.js

```diff
@@ -70,8 +65,12 @@ export const commands = {
   },
 
   makeNumSprites(num, animation) {
-    num = Math.min(num, BIG_NUMBER_GUARD);
-    for (let i = 0; i < num; i++) {
+    // getCappedNumSprites caps num based on number of sprites already created and MAX_NUM_SPRITES
+    const cappedNumSprites = this.getCappedNumSprites(num);
```

Comment:
minor nit but maybe this should be called `maxAllowedNewSprites/getMaxAllowedNewSprites`, or something along those lines?

Reply:
Sounds good - will update.